Synergy/Controllers/RandomCodeController.cs

```diff
@@ -0,0 +1,76 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Synergy.Authorization;
+using Synergy.Models;
+using Synergy.Services;
+using Permissions = Synergy.Authorization.Permissions;
+
+namespace Synergy.Controllers;
+
+[ApiController]
+[Route("/api/[controller]")]
+public class RandomCodeController : ControllerBase
```

Comment:
"RandomCode" doesn't really provide a lot of context about it's use case. I think this would be better as "UserInviteCodeController"

I would also avoid having the route defined using the "[controller]" and just write in the name of the endpoint. Because if you change the name of the controller but don't update your uses then all of the endpoints will break